File: javascript/Baekjoon/Bronze/III/2884.js
// 알람 시계
// https://www.acmicpc.net/problem/2884

let input = require('fs').readFileSync('../../예제.txt').toString().replace(/\r*/g, "").trim().split(' ').map(Number);

if(input[1] < 45) {
    input[0] = (input[0] + 23) % 24;
    input[1] = input[1] + 15;
} else {
    input[1] = input[1] - 45;
}
console.log(input[0], input[1]);

File: Python/beakjoon/Bronze/III/2884.py
# 암람 시계
# https://www.acmicpc.net/problem/2884

data = list(map(int, input().split()))

data[0] = (data[0] + 23) % 24 if data[1] < 45 else data[0]
data[1] = (data[1] + 15) % 60

print(data[0], data[1])